lib/ClassNameBuilder.js

```diff
@@ -8,27 +8,24 @@ export default class ClassNameBuilder {
 
     stringify(block, elem, mods, mixes, cls) {
         return this.b_(block, elem, mods) +
-            (mixes? ' ' + mixes.map(mix => mix?
+            (mixes && mixes.length? ' ' + mixes.map(mix => mix?
                 this.b_(mix.block || block, mix.elem, mix.mods) : '').join(' ') :
                 '') + (cls? ' ' + cls : '');
     }
 
-    joinMixes(mix1, mix2) {
```

Comment:
History says that you named it `joinMixes`, originally it was named as `buildMixes` (https://github.com/bem/bem-react-core/commit/3b9da98b7daae5008087245428ac2baa78a49f7a#diff-168726dbe96b3ce427e7fedce31bb0bcL17)